### DISCUSSION TITLE
From 2025 it will be 15 minutes interval for prices, at least in Sweden.

### DISCUSSION THREAD

**Discussion by tronde-ams** [UtilitechAS/amsreader-firmware#864] [Ideas]
Text in Swedish
https://www.vattenfall.se/fokus/tips-rad/kvartsmatning-och-kvartspris/

I understand it as this will affect all prices from Nordpool?

Major rewrite of code?

**Comment by gskjold** [comment]
Rewrite indeed, but I've seen this one coming, so I have been thinking about how to best solve it at least

**Comment by nossaile** [comment]
On the news today on Radio Sweden:
All electricity trade in Europe will be made per 15-minute intervals and in June most hourly price end-user agreement will automatically be changed to 15-minute based.

**Reply by ArnieO** [reply]
We are aware of this, and will implement increased price granularity.

**Comment by Belaial** [comment]
Been living with 15 minute intervals in Finland (at least Ålands Islands) since 2023 I think even tho Åland follows Sweden's region SE3 when it comes to electricity prices.

**Comment by gskjold** [comment]
Covered in new test firmware in #1032
